tests/plugins/table/headers1row2col.html

```diff
@@ -0,0 +1,32 @@
+<textarea class="none">
```

Comment:
I noticed `getValue()` in the `testHeadersManipulation()` function and started to wonder why you used it instead of `getHtml()`. I think you may just use `<div>`s here and in the `testHeadersManipulation()` function use `getHtml()` instead of `getValue()`.

Reply:
So please update all the test files.